lib/web/apiserver.go

```diff
@@ -308,7 +308,9 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 				}
 			}
 			httplib.SetIndexHTMLHeaders(w.Header())
-			indexPage.Execute(w, session)
+			if err := indexPage.Execute(w, session); err != nil {
+				log.Errorf("failed to execute index page template: %v", err)
```

Comment:
Missing capitalization and punctuation.

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Fixed